crates/re_space_view/src/view_property_ui.rs

```diff
@@ -47,25 +47,55 @@ pub fn view_property_ui<A: Archetype>(
                 .interactive(false)
                 .show_flat(
                     ui,
-                    list_item::PropertyContent::new(display_name).value_fn(|_, ui, _| {
-                        ctx.component_ui_registry.edit_ui(
-                            ctx,
-                            ui,
-                            re_viewer_context::UiLayout::List,
-                            blueprint_query,
-                            blueprint_db,
-                            &blueprint_path,
-                            &blueprint_path,
-                            component_results.get_or_empty(*component_name),
-                            component_name,
-                            &0.into(),
-                        );
-                    }),
+                    list_item::PropertyContent::new(display_name)
+                        .action_button(&re_ui::icons::RESET, || {
+                            ctx.reset_blueprint_component_by_name(&blueprint_path, *component_name);
+                        })
+                        .value_fn(|_, ui, _| {
+                            ctx.component_ui_registry.edit_ui(
+                                ctx,
+                                ui,
+                                re_viewer_context::UiLayout::List,
+                                blueprint_query,
+                                blueprint_db,
+                                &blueprint_path,
+                                &blueprint_path,
+                                component_results.get(*component_name),
+                                *component_name,
+                                &0.into(),
+                            );
+                        }),
                 );
 
-            if let Some(tooltip) = field_info.map(|info| info.documentation) {
-                list_item_response.on_hover_text(tooltip);
-            }
+            let list_item_response =
+                if let Some(tooltip) = field_info.map(|info| info.documentation) {
+                    list_item_response.on_hover_text(tooltip)
+                } else {
+                    list_item_response
+                };
```

Comment:
nit: that is slightly weird/verbose

```suggestion
            if let Some(tooltip) = field_info.map(|info| info.documentation) {
                list_item_response = list_item_response.on_hover_text(tooltip)
            }
```
(needs `list_item_respons` to be `mut`)